src/main/java/games/strategy/engine/data/DelegateList.java

```diff
@@ -33,11 +33,11 @@ public IDelegate getDelegate(final String name) {
     return m_delegates.get(name);
   }
 
-  private void writeObject(final ObjectOutputStream out) {
+  private void writeObject(@SuppressWarnings("unused") final ObjectOutputStream out) {
```

Comment:
This method signature must remain unchanged, as required by the serialization runtime.